requirements.txt

```
@@ -7,4 +7,5 @@ pyopengl==3.1.6
 mediapipe==0.8.11
 keyboard==0.13.5
 scipy==1.9.1
```

```suggestion
scipy~=1.9.1
```
allows you to change the patch number